From c8891f390fe3c593a5990acb0095218afc74d088 Mon Sep 17 00:00:00 2001
From: Tim Parenti <tim@timtimeonline.com>
Date: Tue, 22 Dec 2020 16:05:59 -0500
Subject: [PATCH] Cite publication of Volgograd change

* europe (Europe/Volgograd): Cite pravo.gov.ru.
(Thanks to Stepan Golosunov.)
---
 europe | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/europe b/europe
index fd9e85b..bba4d56 100644
--- a/europe
+++ b/europe
@@ -2900,6 +2900,10 @@ Zone Europe/Astrakhan	 3:12:12 -	LMT	1924 May
 # From Stepan Golosunov (2020-12-05):
 # Currently proposed text for the second reading (expected on December 8) ...
 # changes the date to December 27. https://v1.ru/text/gorod/2020/12/04/69601031/
+#
+# From Stepan Golosunov (2020-12-22):
+# The law was published today on
+# http://publication.pravo.gov.ru/Document/View/0001202012220002
 
 Zone Europe/Volgograd	 2:57:40 -	LMT	1920 Jan  3
 			 3:00	-	+03	1930 Jun 21
-- 
1.8.3.1

